src/patternfly/components/Select/docs/select-icon.md

```diff
@@ -0,0 +1,17 @@
+## Accessibility
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `.pf-c-select` |  accessibility notes. |
+| `aria-hidden="true"` | `.pf-c-select__toggle-icon` | Hides the icon from assistive technologies.
+
+
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-select` | `<div>` |  Initiates a custom select. |
+| `.pf-c-select__toggle` | `<button>` |  Initiates a custom toggle. |
+| `.pf-c-select__toggle-wrapper` | `<div>` |  Initiates a custom select toggle wrapper. |
```

Comment:
"custom select" or just "select"?

Reply:
I think it's called "custom select" because it's the custom component for a select menu (as opposed to `<select>`), but agreed. I think I'll just update it to say _Initiates the select component._